src/main/kotlin/com/vk/admstorm/services/HastebinService.kt

```diff
@@ -3,17 +3,44 @@ package com.vk.admstorm.services
 import com.intellij.openapi.components.Service
 import com.intellij.openapi.components.service
 import com.intellij.openapi.project.Project
-import com.vk.admstorm.CommandRunner
 import com.vk.admstorm.env.Env
+import com.vk.admstorm.env.getByKey
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.jsonPrimitive
+import java.net.URI
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+import java.time.Duration
+import java.util.concurrent.Executors
 
 @Service(Service.Level.PROJECT)
-class HastebinService(private val project: Project) {
+class HastebinService {
```

Comment:
Why don't we add some logs? In case something goes wrong

Reply:
Added